src/decrypt-api/decrypt-api.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
   <PropertyGroup>
-    <Version>0.2.2.0</Version>
+    <Version>0.2.3.0</Version>
```

Comment:
Update also encryptor?

Reply:
Do we always need to bump both together even if no changes were made?

Reply:
yep, it's one tag for both images, we should extract it to a file - I'll do that as part of the CRD (#115)